libbeat/common/transport/httpcommon/httpcommon.go

```diff
@@ -21,7 +21,7 @@ import (
 	"net/http"
 	"time"
 
-	"go.elastic.co/apm/module/apmhttp"
+	"go.elastic.co/apm/module/apmhttp/v2"
```

Comment:
@kvch we will have to port this to https://github.com/elastic/elastic-agent-libs/blob/df72f6afbb56a8f8547523f63fdd5cb0a8671d69/transport/httpcommon/httpcommon.go#L24